Makefile

```diff
@@ -6,9 +6,6 @@ all: list
 build:
 	docker build -t okibot/processors .
 
-install:
```

Comment:
In general, unless the commands aren't standard, I prefer to use the default Python tools. For instance, I prefer running `tox` instead of `make test`.

That's a personal preference, though, so you could argue both ways.